app/utils/sanity/type.go

```diff
@@ -79,15 +100,30 @@ type conflictPair struct {
 
 // checkConflict: check configA vs configB
 // and the value is equal to the conflicts value then complain it
-func (cp *conflictPair) checkConflict() error {
+func (cp *conflictPair) checkConflict(opt ...string) error {
 	if cp.configA.check() &&
 		cp.configB.check() {
-		return fmt.Errorf(" %v conflict with %v", cp.configA.verbose(), cp.configB.verbose())
+		if len(opt) == 0 {
+			return fmt.Errorf(" %v conflict with %v", cp.configA.verbose(), cp.configB.verbose())
```

Comment:
This  `if len(opt) == 0 {`  put outside of  the `if cp.configA.check() &&cp.configB.check()` maybe reduce the judge

Reply:
Only both configA's check and configB's check are true， we will custom the return error to `opt`(now rename to `tips`, `opt` is misleading and meaningless).